enterprise/views.py

```diff
@@ -60,8 +60,11 @@
     filter_audit_course_modes,
 )
 
+from six.moves.urllib.parse import urljoin  # pylint: disable=import-error
 
 logger = getLogger(__name__)  # pylint: disable=invalid-name
+LMS_START_PREMIUM_COURSE_FLOW_URL = urljoin(settings.LMS_ROOT_URL, '/verify_student/start-flow/{course_id}/')
```

Comment:
`PREMIUM` means paid, so `verified` or `professional`. However, I think `premium` would also include `professional-no-id` and I don't think we want to be starting the verification flow for those courses, right?